### ISSUE TITLE
SideNav

### ISSUE THREAD

**Issue by pettinarip** [ethereum/ethereum-org-website#10744]

**Comment by Nagapadmini7** [comment]
I can work on this issue

**Comment by ameeetgaikwad** [comment]
@pettinarip would like to work on this issue

**Comment by pettinarip** [comment]
@ameeetgaikwad pls consider using the [Accordion](https://chakra-ui.com/docs/components/accordion/usage) for this implementation

**Comment by ameeetgaikwad** [comment]
@pettinarip Not able to see much difference in the current implementation and the DS one. anyways here I have used the Accordion from chakra, let me know if you need any changes.

![image](https://github.com/ethereum/ethereum-org-website/assets/76063569/038a8389-7452-4569-9c2c-7a5f86a8d55b)

**Comment by ameeetgaikwad** [comment]
@TylerAPfledderer so sorry, I'll delete that comment.

**Comment by pettinarip** [comment]
Correct. Good point @TylerAPfledderer. This PR will be on hold until we merge the Accordion implementation.

**Comment by F3n4nd0gm** [comment]
https://github.com/ethereum/ethereum-org-website/assets/147960298/494a6cd2-cf19-4eee-9f02-5fb041e35ed4

**Comment by F3n4nd0gm** [comment]
![pettinarip.github.io-develop.zip](https://github.com/ethereum/ethereum-org-website/files/13061848/pettinarip.github.io-develop.zip)

**Comment by wackerow** [comment]
Circling back, this hasn't been forgotten... #10936 has had a couple updates, trying to get that in then will come back to this one.

**Comment by corwintines** [comment]
Removing stale tag since this has a draft PR in progress

**Comment by minimalsm** [comment]
Reassigning this to @pettinarip as he has an open (draft) PR here.
